lib/cog/template_cache.ex

```diff
@@ -1,7 +1,6 @@
 defmodule Cog.TemplateCache do
```

Comment:
I think that's fine; after all, the idea is that with a cache, we'll be saving a lot more calls in the long run.

Having it be its own module is a nice separation of concerns, and makes both the caching logic and the "normal" logic easier to understand.